### DISCUSSION TITLE
386 armv7装不了

### DISCUSSION THREAD

**Discussion by zhangbo1201** [messense/aliyundrive-webdav#451] [Q&A] [answered]
### 问题描述

386 armv7装不了

### 重现步骤

本插件适用于【koolshare merlin armv7l 384】固件平台，你的固件平台不能安装！！！
【2022年06月03日 19:34:55】: 退出安装！

### 版本

386

### 运行平台

ax3000 v2

### 日志

本插件适用于【koolshare merlin armv7l 384】固件平台，你的固件平台不能安装！！！
【2022年06月03日 19:34:55】: 退出安装！

**Comment by messense** [comment, marked as answer]
那就是不支持呗。

**Reply by apple16999** [reply]
我想请问一下，华硕AX3000和AX6000下载任何一个梅林的版本都不能安装，能发开发一个华硕的固件版本啊

**Reply by bacontse** [reply]
哭了😭alist不支持 aliyundrive-webdav也不支持 打算把ax3000v2扔了换个路由

**Reply by messense** [reply]
> 能发开发一个华硕的固件版本啊

无计划。
